shaping/shaper.go

```diff
@@ -47,40 +56,43 @@ const (
 )
 
 // Shape turns an input into an output.
-func Shape(input Input) (Output, error) {
+func (t *HarfbuzzShaper) Shape(input Input) (Output, error) {
```

Comment:
Can you document the sort of errors that can be returned here?

Reply:
@andydotxyz In light of #27, could I instead just drop the error return altogether from this PR?

Reply:
Ah I see. Yes that would work :)